plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
> Upgrading the driver is mandatory if you want to enjoy new game

That's actually really not true. Many new games work great with older drivers, and new drivers frequently have new issues (as well as fixes).

You really cannot assume that everyone has the latest driver.

Even if everyone would want to always have the latest (which is debatable, but let's assume it), it's not always possible. Some people are tied to OEM drivers which don't update frequently. or update with high delay, or stopped updating, and which the reference drivers would refuse to replace, for instance.

Blacklisting or whitelisting based only on the vendor (which is what the code does currently, at least for AMD, but I think also for others) is a recipe for making a lot of incorrect assumptions.